docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -62,3 +63,70 @@ Log in the back office as an Administrator and go to the Users page. Select the
 
 Then log in again into the back office with your new user credentials.
 You have now the possibility to create new Rides and Points of Interest only.
+
+
+
+##  Customize the use forms
+
+We want to have a consistent design over our web site. When using the register form, it was a raw HTML form without any styling on it. We want to welcome our new users with a pretty register page.
```

Comment:
`consistent design in our website`
`When we were using the register form` ... `without any styling` (full stop)